maintainers/maintainer-list.nix

```diff
@@ -417,6 +417,16 @@
     github = "akho";
     githubId = 104951;
   };
+  akkesm = {
```

Comment:
Adding yourself as a maintainer is preferred to be a separate commit.
Something like, `maintainers: add akkesm`. Then apply the passky-desktop commit.